readme.md

```diff
@@ -26,6 +26,7 @@
   * [Can I configure the lighthouse run?](#can-i-configure-the-lighthouse-run)
   * [How does Lighthouse use network throttling, and how can I make it better?](#how-does-lighthouse-use-network-throttling-and-how-can-i-make-it-better)
   * [Are results sent to a remote server?](#are-results-sent-to-a-remote-server)
+  * [How do I get localized Lighthouse results?](#how-do-i-get-localized-lighthouse-results)
```

Comment:
bad link

Reply:
> bad link

haha, how on earth did you notice that :)

thanks!